pmd-core/src/main/java/net/sourceforge/pmd/cli/PMDParameters.java

```diff
@@ -29,105 +29,108 @@
     private boolean benchmark = false;
 
-    @Parameter(names = { "-stress", "-S" }, description = "Performs a stress test.")
+    @Parameter(names = { "--stress", "-stress", "-S" }, description = "Performs a stress test.")
     private boolean stress = false;
 
-    @Parameter(names = "-shortnames", description = "Prints shortened filenames in the report.")
+    @Parameter(names = { "--short-names", "-shortnames" }, description = "Prints shortened filenames in the report.")
     private boolean shortnames = false;
 
-    @Parameter(names = "-showsuppressed", description = "Report should show suppressed rule violations.")
+    @Parameter(names = { "--show-suppressed", "-showsuppressed" }, description = "Report should show suppressed rule violations.")
     private boolean showsuppressed = false;
 
-    @Parameter(names = "-suppressmarker",
+    @Parameter(names = { "--suppress-marker", "-suppressmarker" },
             description = "Specifies the string that marks a line which PMD should ignore; default is NOPMD.")
     private String suppressmarker = "NOPMD";
 
-    @Parameter(names = { "-minimumpriority", "-min" },
+    @Parameter(names = { "--minimum-priority", "-minimumpriority", "-min" },
             description = "Rule priority threshold; rules with lower priority than configured here won't be used. "
                     + "Valid values are integers between 1 and 5 (inclusive), with 5 being the lowest priority.",
             validateValueWith = RulePriorityValidator.class)
     private int minimumPriority = RulePriority.LOW.getPriority();
 
-    @Parameter(names = { "-property", "-P" }, description = "{name}={value}: Define a property for the report format.",
+    @Parameter(names = { "--property", "-property", "-P" }, description = "{name}={value}: Define a property for the report format.",
             converter = PropertyConverter.class)
     private List<Properties> properties = new ArrayList<>();
 
-    @Parameter(names = { "-reportfile", "-r" },
+    @Parameter(names = { "--report-file", "-reportfile", "-r" },
                description = "Path to a file to which report output is written. "
                    + "The file is created if it does not exist. "
                    + "If this option is not specified, the report is rendered to standard output.")
     private String reportfile = null;
 
-    @Parameter(names = { "-version", "-v" }, description = "Specify version of a language PMD should use.")
+    @Parameter(names = { "--use-version", "-version", "-v" }, description = "Specify version of a language PMD should use.")
     private String version = null;
 
-    @Parameter(names = { "-language", "-l" }, description = "Specify a language PMD should use.")
+    @Parameter(names = "--version", description = "Display current version of PMD.", help = true)
+    private boolean currentVersion = false;
+
+    @Parameter(names = { "--language", "-language", "-l" }, description = "Specify a language PMD should use.")
```

Comment:
If we do introduce `--use-version` as per #3426, I don't think we should introduce `--language` as the option will be deprecated anyway